config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+ISDN1jKTGDsf91DLMWjozmKAZAhDzV9OOPsI+rZgPdg5Nt6aVBJDA7jLlGsfCYqticLVp+/4brgsmdgpPhBJS/UrErPTDNY9b05HNUIBhfcAQ86QpncjRvRzlZu8VGaeuwWDDQ/p4pccHcswI+1v2oJG5C2iCZclUKhPW0z+EHiqv+1TmgT+vgZpbGxWQoODtSgeSM7xbf/p5N8C8WH8z5l2A4WE4hdfEqN0vr32vH8uCzl2M4B1hgGc/QFKSIuT2qi4BW2wRWw7vbIaBoJgJxz04slN2TKDaE52hx/s37Bafpet1kW0C1aW7fZN2BZlSjcHSkF6kHMMTYkTlWnCN1v4dHdTb2t7StzP89aAXcpNLXN/E3ISxo+k4ZOCXQeEtkC2lq7LuB8ZPAlq1pMssHWQlk+92Pt7AAKs--1MiOy4keiVeyswCw--jru7cR2e0KN8bJqGXKOwag==
```

Comment:
Need to remove this file that was generated by Rails, and do something with it.

@matthewd is it needed for the app to run? Can I just `rm` it?

Reply:
Turns out this file isn't as bad as I thought:

https://www.engineyard.com/blog/rails-encrypted-credentials-on-rails-5.2

I rekon we need to `rm` it and have a step in the Readme that's like:

"You'll need to generate a session token" or something.

I thought about disabling sessions, but we need them for CSRF which we may use for search forms later on.

Reply:
We shouldn't need CSRF for search forms though? Should be able to just disable?

Reply:
If the we have a search controller on our end we may want to do POST's instead of GET to stop it from being smashed by some bot or what ever.

Also disabling sessions/cookies in Rails seems like a lot of effort - I don' think it's something worth doing - and if we ever want to do anything session-ish later on, we'll have that super power.

Reply:
🤷🏼‍♂️👍🏼